packages/admin-panel/src/table/DataFetchingTable/FilterCell.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { DefaultFilter } from '../columnTypes/columnFilters';
+
+const FilterWrapper = styled.div`
```

Comment:
```suggestion
const FilterWrapper = styled.search`
```

[`<search>` comes with implicit `search` ARIA role](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/search), and in older browsers that don’t support it, it gracefully falls back to a `div`